src/main/java/frc/trigon/robot/poseestimation/t265/T265Constants.java

```diff
@@ -0,0 +1,7 @@
+package frc.trigon.robot.poseestimation.t265;
+
+import frc.trigon.robot.poseestimation.poseestimator.PoseEstimatorConstants;
+
+public class T265Constants {
+    public static final PoseEstimatorConstants.StandardDeviations STANDARD_DEVIATIONS = new PoseEstimatorConstants.StandardDeviations(0.0000001, 0.0000001);
```

Comment:
Why is this here and not in PoseEstimatorConstants?

Reply:
Because it's the stddevs of the T265, not the pose estimator

Reply:
But it's only used in the post estimator. For this, add t265 to the name

Reply:
It's the pose estimator constants, not the pose estimator and t265. Camera constants also don't go into the pose estimator's constants.